src/main/resources/hudson/plugins/scm/koji/KojiChangeLogSet/digest.jelly

```diff
@@ -9,7 +9,18 @@
                 <j:forEach var="cs" items="${it.items}" varStatus="loop">
```

Comment:
you've reverted it back to `cs`. What does `cs` mean? It's not clear to me.

Reply:
cs means changeset. Judovana wanted wanted me to rename it back to cs.

Reply:
if it means changeset, can we name it `changeset`?